src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { Fragment } from 'react';
+import SimpleBarChart from '@carbon/charts-react/bar-chart-simple';
+import StackedBarChart from '@carbon/charts-react/bar-chart-stacked';
+import GroupedBarChart from '@carbon/charts-react/bar-chart-grouped';
+import '@carbon/charts/styles.css';
+import classnames from 'classnames';
+import every from 'lodash/every';
+import isNil from 'lodash/isNil';
+
+import { BarChartCardPropTypes, CardPropTypes } from '../../constants/PropTypes';
+import {
+  CARD_SIZES,
+  BAR_CHART_TYPES,
+  BAR_CHART_ORIENTATION,
+} from '../../constants/LayoutConstants';
+import Card from '../Card/Card';
+import { settings } from '../../constants/Settings';
+
+const { iotPrefix } = settings;
+
+const BarChartCard = ({
+  title,
+  content: {
+    xLabel,
+    yLabel,
+    orientation = BAR_CHART_ORIENTATION.VERTICAL,
+    chartType = BAR_CHART_TYPES.SIMPLE,
+    data,
+    isTimeSeries = false,
+  },
+  size,
+  locale,
+  i18n,
+  isExpanded,
+  isLazyLoading,
+  className,
+  ...others
+}) => {
+  const isAllValuesEmpty = data
```

Comment:
is a tertiary necessary here? I am not too familiar with this lodash function but it looks to return true or false based off the check.

Reply:
true, simplified, thanks!